grafonnet/dashboard.libsonnet

```diff
@@ -149,6 +150,7 @@ local timepickerlib = import 'timepicker.libsonnet';
     addLink(link):: self {
       links+: [link],
     },
+    addDashboardLinks(dashboardLinks):: std.foldl(function(d, t) d.addLink(t), dashboardLinks, self),
```

Comment:
Seems to me `addLinks` would be a more appropriate name. The added "dashboard" segment seems repetitive since it's being called on a dashboard...`dashboard.new(...).addDashboardLinks([...])`. And it's intended to iterate over an array calling `addLink`. These should have the same name except plural - similar to how we have `addTemplate`/`addTemplates`, `addPanel`/`addPanels`, etc.